packages/flatseal.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Flatseal < Package
+  description 'Flatseal is a graphical utility to review and modify permissions from your Flatpak applications.'
+  homepage 'https://github.com/tchx84/Flatseal'
+  version '1.6.8'
+  license 'GPL-3+'
+  compatibility 'all'
+  source_url 'https://github.com/tchx84/Flatseal/archive/v1.6.8.tar.gz'
+  source_sha256 '28d05995effa2858483283cbc9fb54c55bf4dcb6267e4d7d495a32ff724806b1'
+
+  depends_on 'libhandy'
+  depends_on 'desktop_file_utils'
+  depends_on 'gjs'
+  depends_on 'gtk3'
+
+  def self.build
+    system "meson #{CREW_MESON_OPTIONS} builddir"
+    system "samu -C builddir"
+  end
+
+  def self.check
+    system "samu -C builddir test"
+  end
+
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+    FileUtils.symlink 'com.github.tchx84.Flatseal', "#{CREW_DEST_PREFIX}/bin/flatseal"
```

Comment:
Shouldn't `'com.github.tchx84.Flatseal'` have a path?  The working directory is different than the actual installed directory.

Reply:
#TodayILearned, when making a symlink with FileUtils or `ln`, the first argument is where the symlink should point and it can be *absolute or relative*. So this just makes a symlink called `flatseal` at `/usr/local/bin` that points to `com.github.tchx64.Flatseal`. It could also be
```ruby
FileUtils.symlink './com.github.tchx84.Flatseal', "#{CREW_DEST_PREFIX}/bin/flatseal"
```
but that's just extraneous.

Reply:
No, I mean the destination directory for `com.github.tchx84.Flatseal` is somewhere under `CREW_PREFIX`, not `CREW_DEST_DIR`.  Was this even tested?

Reply:
This was tested and it works.

Reply:
I'm not sure how this would work since the symlink doesn't include a path.

Reply:
The first argument to FileUtils.symlink is the path. `./` is assumed.

Reply:
I don't think you understand.  That is not a path, that is just relative, not absolute.

Reply:
Yes, and it works. Test it yourself and see if you really like.

Reply:
In this case you got lucky and it works because `com.github.tchx84.Flatseal` ends up in the bin directory but I'm sure that is not always the case for most packages.

Reply:
...

🤦🏻